app/src/main/res/layout/character_detail_layout.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
where do you use this layout?

Reply:
As i told Braian that xml was created to be used in the next commit, in which the functionality of the fd will be done. To avoid confusion, it is currently deleted.

Reply:
Remember that we only put in a PR things that we use at that moment. Never things that we will use in the future